monkeys/users.json

```diff
@@ -20,6 +20,56 @@
           "id": "33ec2e20-9bed-44b2-b891-a1dbaf3da158"
         }
       ]
+    },
+    {
+      "api": "https://nginz-https.bella.wire.link",
+      "webSocket": "https://nginz-ssl.bella.wire.link",
+      "blackList": "https://clientblacklist.wire.com/staging",
+      "teams": "https://teams.bella.wire.link",
+      "accounts": "https://account.bella.wire.link",
+      "website": "https://wire.com",
+      "title": "bella.wire.link",
+      "passwordForUsers": "Aqa123456!",
+      "domain": "bella.wire.link",
+      "users": [
+        {
+          "id": "37a27c29-398b-44c6-9334-79cf84024f74",
+          "email": "augusto.c.dias+test1@wire.com"
+        },
+        {
+          "id": "56f06ac1-69cb-43ab-96f1-816f6ddd6edb",
+          "email": "augusto.c.dias+test3@wire.com"
+        },
+        {
+          "id": "691e3c09-45e1-4845-89f4-121fef480ec8",
+          "email": "augusto.c.dias+test2@wire.com"
+        }
+      ]
+    },
+    {
+      "api": "https://nginz-https.chala.wire.link",
+      "webSocket": "https://nginz-ssl.chala.wire.link",
+      "blackList": "https://clientblacklist.wire.com/staging",
+      "teams": "https://teams.chala.wire.link",
+      "accounts": "https://account.chala.wire.link",
+      "website": "https://wire.com",
+      "passwordForUsers": "Aqa123456!",
+      "domain": "chala.wire.link",
+      "title": "chala.wire.link",
```

Comment:
Why do we hardcode the backend urls? You know we can query then from: 

https://nginz-https.anta.wire.link/deeplink.json
https://nginz-https.bella.wire.link/deeplink.json
https://nginz-https.chala.wire.link/deeplink.json